cmd/sys-mgmt-agent/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN make cmd/sys-mgmt-agent/sys-mgmt-agent
 RUN make cmd/sys-mgmt-executor/sys-mgmt-executor
 
 # Get the Docker-in-Docker image layered-in now.
-FROM docker:20.10.7
+FROM docker:20.10.8
```

Comment:
would it be better to use `docker:stable`?

Reply:
@lenny-intel  ?

Reply:
Getting really tired of bumping this container.

Reply:
I don't see a `stable` tag. Also typically we nee to pin it to an actual version.
https://hub.docker.com/_/docker?tab=tags&page=1&ordering=last_updated

Reply:
here is the stable tag, and it's old right now
https://hub.docker.com/layers/docker/library/docker/stable/images/sha256-ce46ea18798c578f8923e0ced4c9996c30363fc35edd1b1c40e5e542407818ac?context=explore